app/src/main/java/br/com/useblu/oceands/client/ui/home/HomeActivity.kt

```diff
@@ -110,9 +114,16 @@ class HomeActivity : AppCompatActivity() {
 
             if (showBottomSheet.value) {
                 OceanBottomSheet(
-                    content = {
-                      Text(text = "Exemplo simples de bottom sheet")
-                    },
+                    model = OceanBottomSheetModel(
+                        title = "Title",
+                        message = "Message",
+                        subMessage = "SubMessage",
+                        imageUrl = "https://portal-cicloentrada.blu.com.br/assets/icons/coin_trail-cc541831a7fbf4d215f3910fb241b14701f5ab0f79d574ad3a6e12379b7e871e.png",
+                        code = 2000,
+                        actionPositive = "Aceitar" to {},
+                        actionNegative = "Cancelar" to {},
+                        buttonsOrientation = BottomSheetButtonsOrientation.Vertical
+                    ),
                     showBottomSheet = showBottomSheet
```

Comment:
Será que não conseguimos remover a necessidade desse parametro showBottomSheet...Haja em vista que quanto invocarmos o componente OceanBottomSheet já deveria obrigatoriamente ser exibido

Reply:
a lógica pra invocar o componente é diferente dos componentes xml
precisa ter algum modo de controle pro compose passar pela função da Modal quando a gente quiser que a modal seja exibida; e também para que na recomposição a modal não seja mais exibida quando for o caso

Reply:
aqui não existe só o "dismiss" ou o "show" que tinhamos antes. precisa ter uma condicional la na implementação pra nas composições a gente exibir ou não a bottom sheet

Reply:
Mas o if que tava envolvendo esse componente era desnecessário, atualizei removendo ele aqui